examples/gum/hook_open/Cargo.toml

```diff
@@ -12,3 +12,5 @@ crate-type = ["cdylib"]
 frida-gum = { path = "../../../frida-gum"}
 lazy_static = "1.4"
 ctor = "0.1"
+libc = "*"
```

Comment:
Please pin a libc version here because example executables do not have a `Cargo.lock`